src/helm/config/model_metadata.yaml

```diff
@@ -593,16 +593,32 @@ models:
     tags: [] # TODO: add tags
 
   - name: google/gemini-pro
-    display_name: Gemini
-    description: Gemini is a multimodal model able to reason across text, images, video, audio and code. ([paper](https://arxiv.org/abs/2312.11805))
+    display_name: Gemini Pro
+    description: Gemini Pro is a multimodal model able to reason across text, images, video, audio and code. ([paper](https://arxiv.org/abs/2312.11805))
+    creator_organization_name: Google
+    access: limited
+    release_date: 2023-12-13
+    tags: [TEXT_MODEL_TAG, GOOGLE_GEMINI_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG]
+
+  - name: google/gemini-1.0-pro-001
+    display_name: Gemini 1.0 Pro
+    description: Gemini 1.0 Pro is a multimodal model able to reason across text, images, video, audio and code. ([paper](https://arxiv.org/abs/2312.11805))
```

Comment:
Should we add a disclaimer here to let people know that this model is not fixed and can be used to get the latest (I suppose) Gemini Pro model